packages/backend/template.yaml

```diff
@@ -247,6 +268,54 @@ Resources:
       FunctionName: !Ref ChangeStatusFunction
       Principal: apigateway.amazonaws.com
 
+  #----------------------------------------#
+  # Lambda: CheckCaptcha from Websocket
+  #----------------------------------------#
+  AxiosDepLayer:
+    Type: AWS::Serverless::LayerVersion
+    Properties:
+      LayerName: axios-dependency
+      Description: Axios dependency
+      ContentUri: axios-layer/
+      CompatibleRuntimes:
+        - nodejs14.x
+      RetentionPolicy: Retain
+    Metadata:
+      BuildMethod: nodejs14.x
+  CheckCaptchaFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: checkcaptcha/
+      Handler: app.handler
+      MemorySize: 256
+      Runtime: nodejs14.x
+      Layers:
+        - !Ref AxiosDepLayer
+      Environment:
+        Variables:
+          SOCKET_TABLE_NAME: !Ref SocketTableName
+      Policies:
+      - DynamoDBCrudPolicy:
+          TableName: !Ref SocketTableName
+      - Statement:
+        - Effect: Allow
+          Action:
+          - 'secretsmanager:GetSecretValue'
+          Resource:
+          - !Sub 'arn:aws:secretsmanager:${AWS::Region}:${AWS::AccountId}:secret:nouns/GoogleApiKey-*'
+          - !Sub 'arn:aws:secretsmanager:${AWS::Region}:${AWS::AccountId}:secret:nouns/GoogleProjectId-*'
+          - !Sub 'arn:aws:secretsmanager:${AWS::Region}:${AWS::AccountId}:secret:nouns/ReCaptchaKey-*'
+          - !Sub 'arn:aws:secretsmanager:${AWS::Region}:${AWS::AccountId}:secret:nouns/ReCaptchaAction-*'
+          - !Sub 'arn:aws:secretsmanager:${AWS::Region}:${AWS::AccountId}:secret:nouns/ReCaptchaThreshold-*'
```

Comment:
Update with the fewer keys per above

Reply:
It's now provided as ENV variable to lambda function